articles/azure/azs-how-setup-recovery-portal.md

```diff
@@ -0,0 +1,486 @@
+
+1. Under **Protected items** in the vault select **Replicated items**.
+
+2. Select **Replicate**.
+
+3. In Source, select **On-premises**.
+
+4. In **Source location** select the configuration server you set up.
+
+5. In Machine type, select Physical machines.
+
+6. Select the process server (configuration server).
+
+    ![Azure replication source](images/azs-browser-replication-source.png)
+
+7. In Target, select the **subscription** and the **Post-failover resource group** in which you want to create the VMs after failover. Choose the **Post-failover deployment model** that you want to use for the failed-over VMs.
+
+8. Select the Public Azure storage account in which you want to store replicated data or create a new storage account.
+
+9. Select **Configure now** for selected machines to apply the network setting to all machines you select for protection. Select **Configure later** if you want to select the Azure network separately for each machine.
+
+10. Select or create a new Azure network and subnet to which Azure VMs connect when they're created after failover.
+
+    ![Azure replication target](images/azs-browser-replication-target.png)
+
+11. In Physical Machines, select **Physical machine**. Specify the name, IP address and the operating system of each machine you want to replicate, you should have noted these down from when you set up the machine prerequisites.
+
+    * Use the internal IP address of the machine.
+
+    **If you specify the public IP address replication might not work.**
+
+    ![Azure replication machine](images/azs-browser-replication-machine.png)
+
+12. In Properties > Configure properties, select the account that the process server will use to automatically install Mobility Service on the machine. If the configuration server has yet to sync with Public Azure, no account will appear here.
+
+    ![Azure replication properties](images/azs-browser-replication-properties.png)
+
+13. In Replication settings > Configure replication settings, check that the correct replication policy is selected.
+
+    ![Azure replication settings](images/azs-browser-replication-settings.png)
+
+14. Select **Enable Replication**.
+
+15. Track progress of the Enable Protection job in Settings > Jobs > Site Recovery Jobs. After the Finalize Protection job runs, the machine is ready for failover.
+
+    * The VM will need to synchronize before a failover can take place. You can see synchronization progress of the VM in the vault.
+
+## Step 6: Run a disaster recovery drill
+
+You can run a test failover to Azure to make sure that everything's working as expected. This failover won't affect your production environment.
+
+### Verify machine properties
+
+Before you run a test failover, verify the machine properties, and make sure that they comply with Azure requirements. You can view and modify properties as follows:
+
+1. In Protected Items, select Replicated Items > VM.
```

Comment:
In the *Protected* blade, ...

(if it's a blade, otherwise specify what it is)

Reply:
done